drange/lib/DataStructure.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
```

Comment:
Is this original code? Either way, I would prefer having it in a separate npm package (i.e. no no-jsverify code in this repository).

Also as a side-note, even we don't generate discontinuous-ranges, we (read: *I*) prefer immutable data-structures. And tested ones.